frontend/test/metabase-visual/dashboard/filter-sidebar.cy.spec.js

```diff
@@ -0,0 +1,33 @@
+import { restore } from "__support__/e2e/cypress";
+
+const questionDetails = {
+  query: { "source-table": 2 },
+};
+
+describe("visual tests > dashboard > filter sidebar", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("renders correctly", () => {
+    cy.createQuestionAndDashboard({ questionDetails }).then(
+      ({ body: { card_id, dashboard_id } }) => {
+        cy.visit(`/dashboard/${dashboard_id}`);
+
+        openFilterSidebar();
+
+        cy.findByText("Label");
+
+        cy.percySnapshot("Shows dashboard filter sidebar " + Date.now());
```

Comment:
We can easily leave only this part in the test itself and move the setup in the beforeEach hook.

Reply:
👍 Done